todoapp.js

```diff
@@ -0,0 +1,19 @@
+document.addEventListener("DOMContentLoaded", () => {
+  let sub = document.querySelector("#submit");
+  let list = document.getElementById("list");
+
+  let input = document.getElementById("things");
+
+  sub.addEventListener("click", event => {
+    let create_li = document.createElement("li");
+    if (input.value === "") {
+      // let create_par = document.createElement("p");
+      error.innerHTML = "<p>Input somethin'</p>";
```

Comment:
Nice job handling empty inputs. If you wanted to account for a string made up of empty spaces (i.e. "      "), look into the String.prototype.trim() method!